app/mailers/report_mailer.rb

```diff
@@ -3,13 +3,19 @@ class ReportMailer < ApplicationMailer
   # TODO: change this for production
   default to: 'lihan@adhocteam.us'
 
-  def year_to_date_report_email(_report)
-    # TODO: add report as s3 upload
-    # attachments['report.csv'] = report
+  def year_to_date_report_email(report_file)
+    s3 = Aws::S3::Resource.new(
```

Comment:
I like to :bike: :house: a lot when it comes to reviews - this might be nice in an `s3` or `s3_client` private method.